_inc/client/security/antispam.jsx

```diff
@@ -87,12 +85,33 @@ export const Antispam = withModuleSettingsFormHelpers(
 			};
 			let akismetStatus = '',
 				foldableHeader = __( 'Checking your spam protection…', 'jetpack' ),
-				explanation = true;
+				explanation = jetpackCreateInterpolateElement(
+					__(
+						"If you don't already have an API key, then <a>get your API key here</a>, and you'll be guided through the process of getting one.",
+						'jetpack'
+					),
+					{
+						a: <a href={ 'https://akismet.com/wordpress/' } />,
+					}
+				);
 
 			if ( null === this.props.isAkismetKeyValid ) {
 				textProps.value = __( 'Fetching key…', 'jetpack' );
 				textProps.disabled = true;
 				explanation = false;
+			} else if (
+				! this.props.isDirty() &&
+				this.props.getSettingCurrentValue( 'wordpress_api_key' ) === '' &&
+				this.props.isAkismetKeyValid
+			) {
+				textProps.value = __( "A valid key has been set in your site's configuration." );
```

Comment:
Could you add a textdomain here?

```suggestion
				textProps.value = __( 'A valid key has been set in your site’s configuration.', 'jetpack' );
```

Reply:
Good catch! Added in d3e27c6 👍